OxfordBodleian/Juel-Jensen/BDLaethd18.xml

```diff
@@ -37,18 +39,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <physDesc>
                         <bindingDesc>
                             <binding contemporary="true">
-                                <decoNote xml:id="b1">Two wooden boards covered with orange tooled leather (full cover); textile inlays.
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with orange tooled <material key="leather"/>.
                                 </decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
                                     <material key="wood"/>       
                                     <material key="leather"/>     
-                                    <material key="textile"/>                               
-                                </decoNote>
-                                <decoNote xml:id="b3" type="SewingStations">4</decoNote>
-                                <decoNote xml:id="b5">Blind tooled ornament on the front and back covers and on the inner side of the leather cover
-                                    (circles with dots, straight lines, interlaced line fillet, double crescents with dots, criss-cross, a cross). 
-                                    <!-- Description of the tooled ornamentation might need some revision -->
+                                    <material key="textile"/>    
                                 </decoNote>
+                                <decoNote xml:id="b3" type="Cover" color="orange">Full cover decorated with blind tooled ornament
+                                    (circles with dots, straight lines, interlaced line fillet, double crescents with dots, criss-cross, a cross). The ornamentation decorates both the
+                                   inner and outer sides of the cover.
```

Comment:
The front and back covers are decorated with blind tooled ornaments. The covers are decorated with circular and linear elements that form cross- or square-shaped patterns and feature a central cross within two rectangular frames respectively decorated with interlace and a criss-cross motifs.